### ISSUE TITLE
Question: should this plugin be installed with the lspconfig?

### ISSUE THREAD

**Issue by dustinboston** [jose-elias-alvarez/typescript.nvim#65]
I'm still early in my transition to the neovim ecosystem of plugins.
I have typescript configured with the lspconfig settings and it seems to be working.

```
lspconfig.tsserver.setup {
    capabilities = capabilities,
    on_attach = on_attach
}
```

However, most of the files I'm working with look like this:

![image](https://user-images.githubusercontent.com/647464/225954759-c7af521f-6884-481c-9675-a00a7eaea129.png)

I also have treesitter installed and typescript configured there as well.
So I thought it might help to install typescript.nvim to get better language support.
Is this right?

**Comment by jose-elias-alvarez** [comment]
Yes, this plugin depends on nvim-lspconfig but has a [different setup method](https://github.com/jose-elias-alvarez/typescript.nvim#setup). You should follow the instructions there and make sure you are no longer calling `lspconfig.tsserver.setup` anywhere in your config.

As for your highlighting issues - it looks like it has something to do with your treesitter config and / or colorscheme, neither of which is LSP-related.
